StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingSaveToLinkVerification/NetworkingSaveToLinkBodyView.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  NetworkingSaveToLinkBodyView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 2/14/23.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+@available(iOSApplicationExtension, unavailable)
+protocol NetworkingSaveToLinkVerificationBodyViewDelegate: AnyObject {
+    func networkingSaveToLinkVerificationBodyView(
+        _ view: NetworkingSaveToLinkVerificationBodyView,
+        didEnterValidOTPCode otpCode: String
+    )
+}
+
+@available(iOSApplicationExtension, unavailable)
+final class NetworkingSaveToLinkVerificationBodyView: UIView {
+
+    weak var delegate: NetworkingSaveToLinkVerificationBodyViewDelegate?
+
+    private(set) lazy var otpTextField: UITextField = {
+       let textField = InsetTextField()
+        textField.textColor = .textPrimary
+        textField.placeholder = "OTP"
+        textField.keyboardType = .numberPad
+        textField.layer.cornerRadius = 8
+        textField.layer.borderColor = UIColor.textBrand.cgColor
+        textField.layer.borderWidth = 2.0
+        textField.addTarget(
+            self,
+            action: #selector(otpTextFieldDidChange),
+            for: .editingChanged
+        )
+        NSLayoutConstraint.activate([
+            textField.heightAnchor.constraint(equalToConstant: 56)
+        ])
+
+        return textField
+    }()
+
+    init(email: String) {
+        super.init(frame: .zero)
+        let verticalStackView = UIStackView(
+            arrangedSubviews: [
+                otpTextField,
+                CreateEmailLabel(email: email),
+            ]
+        )
+        verticalStackView.axis = .vertical
+        verticalStackView.spacing = 20
+        addAndPinSubview(verticalStackView)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    @objc private func otpTextFieldDidChange() {
+        guard let otp = otpTextField.text else {
+            return
+        }
+
+        if otp.count == 6 && Int(otp) != nil {
```

Comment:
should we check if all characters are digits? otherwise users can enter "-12345" and it would be a valid int. Doesn't really change that much since backend will fail but probably better to catch on the client?

Reply:
yea this is just temporary, the OTP UI will change by introducing a reusable OTP component, which will maybe (or maybe not) provide us with new validation